Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -373,7 +373,7 @@ void BluetoothReal::WaitForHCICommandComplete(const u16 opcode)
   for (int tries = 0; tries < 100; ++tries)
   {
     if (libusb_interrupt_transfer(m_handle, HCI_EVENT, buffer.data(),
-                                  static_cast<int>(buffer.size()), &actual_length, 20) == 0 &&
+                                  static_cast<int>(buffer.size()), &actual_length, TIMEOUT) == 0 &&
```

Comment:
Dolphin can wait for any arbitrary duration, so you definitely could change the timeout value here. However, the only reason to do that is if an adapter is going to be used over the network, which is not a supported usage anyway (because games expect the adapter to be local).